images/linux/scripts/installers/go.sh

```diff
@@ -6,53 +6,24 @@
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
-golangTags="/tmp/golang_tags.json"
+# Fail out if any setups fail
+set -e
 
-# This function installs Go using the specified arguments:
-#   $1=MajorVersion (1.11)
-#   $2=IsDefaultVersion (true or false)
-function InstallGo () {
-    version=$( getFullGoVersion $1 )
-    downloadVersion="go$version.linux-amd64.tar.gz"
-    goFolder="$AGENT_TOOLSDIRECTORY/go/$version/x64"
+toolsetJson="$INSTALLER_SCRIPT_FOLDER/toolset.json"
+toolsetVersions=(`ls $AGENT_TOOLSDIRECTORY/go`)
```

Comment:
Should we get the list of versions from $toolsetJson?

Reply:
Done.